docs/modules/user-guide/partials/proc_migrating-plug-ins.adoc

```diff
@@ -16,21 +16,16 @@ java8.yaml
 ---
```

Comment:
The current plan is to remove plugin support from the spec (https://github.com/devfile/api/pull/333). We will need to revisit this migration guide to reflect that.

Reply:
So major changes made to the doc now?  Or later? Since the doc's focus is on migrating plugins and we're removing support for plugins, won't most of the doc's content have to be removed?

Reply:
Most of the content on this page does not apply anymore since we are removing the `plugin` support. However, I am not sure how we wanted to position it in this migration doc. @davidfestal @l0rd How do you want to position this on the migration doc?

Reply:
Yes, @davidfestal and/or @l0rd, please let me know what you think, and I'll make doc changes accordingly.

Reply:
We should mention that in devfile v2:

1. Parents allow to reuse an existing devfile and (partially) replace plugins functionality  
2. Che IDEs and their tooling can be specified as annotations in a v2 devfile or in a distinct yaml file (`.che/che-editor.yaml` and `.che/che-theia-plugin.yaml`).

A few inks:
- [Devfile plugins are not suited to specify che-theia plugins](https://github.com/eclipse/che/issues/18669)
- [Che Editors and Che Theia plugins support in Devfile v2](https://github.com/eclipse/che/issues/18668)
- [Plugins replacement · Issue #364 · devfile/api](https://github.com/devfile/api/issues/364)

Reply:
Thanks, guys! I've addressed this comment in my recent commit, `added info on more devfile v2 features` so I'm resolving this conversation.